db/schema.rb

```diff
@@ -10,15 +10,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_09_07_095906) do
+ActiveRecord::Schema.define(version: 2020_09_08_110952) do
 
   create_table "bookings", force: :cascade do |t|
     t.integer "user_id"
     t.integer "host_id"
     t.datetime "start_time"
     t.datetime "end_time"
     t.datetime "confirmed_at"
-    t.integer "confirmed_by"
+    t.integer "confirmed_by_id"
     t.string "state"
```

Comment:
I'd suggest storing state as an integer to allow flexibility with name changing without having to migrate data. We could then also use [Active Records Enum](https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html) in the Booking model which would add scopes such as `Booking.confirmed` and allow the calling of `confirmed?` on a booking without any additional code.